source/blog/_posts/2016-01-26-shopware-environment.md

```diff
@@ -0,0 +1,143 @@
+---
+draft: true
+title: Shopware Environment
+tags:
+    - environment
+    - htaccess
+    - vhost
+
+categories:
+- dev
+github_link: blog/_posts/2016-01-26-shopware-environment.md
+
+authors: [bc]
+---
+
+Shopware is able to handle multiple environments using the `SHOPWARE_ENV` environment variable.
+
+## Setting the environment
+
+### Apache
+
+You can set the environment for your local installation using the `.htaccess`:
+
+```
+SetEnv SHOPWARE_ENV dev
+```
+
+This also can be done conditionally for a specifix Host or Subshop:
+
+```
+SetEnvIf Host "dev.shopware.in" SHOPWARE_ENV=dev
+```
+
+On my local development machine I configured it in the apache vhost configuration for all installations:
+
+```
+<VirtualHost *:80>
+    DocumentRoot "/home/bcremer/www/"
+    SetEnv SHOPWARE_ENV dev
+</VirtualHost>
+```
+
+### Nginx
+When using the [shopware-with-nginx](https://github.com/bcremer/shopware-with-nginx) configuration:
```

Comment:
ah ok, didn't understand your original intention with it, and assumed it was just some copy paste gone wrong